src/components/custom/styles/selectStyles.js

```diff
@@ -94,7 +94,7 @@ const generalStyle = {
     textAlign: "left",
     border: "transparent",
```

Comment:
theme.palette.transparent.main
```suggestion
    border: theme.palette.transparent.main,
```